### DISCUSSION TITLE
EPISIM with input data from another trafic simulator

### DISCUSSION THREAD

**Discussion by e-elarbi** [matsim-org/matsim-episim-libs#83] [Q&A]
Hi,
I'm a working on trafic and propagation simulation, recently I am trying to let the matsim-episim works for me, it worked with a predefined Berlin scenario, my goal is to get data about population and event files from another trafic simulator, adapt it to matsim syntax and give it as input to EPISIM, but the question I have here if this can work or not ? If it will take me short time or not ? if there is other issues I can not see being someone who haven't worked well with Matsim

**Comment by rakow** [comment]
Hello,

one important point is the quality of the input data. Complete daily plans, i.e. activity chains with activity type, location and duration, are required. Everything else that is probably output of your traffic simulation like vehicle movements is not needed.

You would need to convert your data into a MATSim event file. There are library functions to achieve that without too much effort. If you know java programming, I presume it won't be difficult.  Episim basically only uses MATSim file formats, and extensive MATSim knowledge is not required.

For a realistic scenario, you need a lot more additional data: What restrictions have been in place by the government? How was the activity participation over the time? How are the vaccination and testing rates? Or even how was the weather (leading to activities performed outside).

All these aspects can be modelled with episim and require the data and some work to integrate and calibrate it.
I presume converting your existing data (if it has the necessary attributes) and setting up a simple scenario should not consume too much time. On the other hand, modelling and calibrating a realistic scenario will be something more time-consuming.

**Reply by e-elarbi** [reply]
Thank you very much @rakow for the answer and details. I will make a try and I'll come back to you for further (I hope not) questions. 

Best regards.
